src/main/java/com/code_intelligence/jazzer/driver/Opt.java

```diff
@@ -150,7 +154,7 @@ public final class Opt {
   public static final List<String> instrumentOnly = stringListSetting("instrument_only", ',',
       "Comma separated list of jar files to instrument. No fuzzing is performed.");
 
-  static final boolean mergeInner = boolSetting("internal.merge_inner", false, null);
+  public static final boolean mergeInner = boolSetting("internal.merge_inner", false, null);
```

Comment:
Is this change needed?

Reply:
This is unfortunately needed. It is not being considered the same package for access when we extract the bootstrap jar and inject it into the bootstrap. Maybe the new "only load the Opt class in through system classloader" PR will fix this though. I'll check when I pull that change in and retest.

Reply:
It's fine if it works, but it's also worrying: While I can see why Opt would be loaded in the bootstrap class path, I wouldn't expect anything that reads this property to be loaded in a different class loader.

Reply:
after pulling in the changes to Opt, this is no longer needed

Reply:
not sure exactly why, but I'm also not digging in too much